LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentNonce.kt

```diff
@@ -106,11 +106,11 @@ data class LocalPaymentNonce internal constructor(
                 clientMetadataId = clientMetadataId,
                 billingAddress = billingAddress,
                 shippingAddress = shippingAddress,
-                givenName = givenName ?: throw JSONException("givenName is null"),
-                surname = surname ?: throw JSONException("surname is null"),
-                phone = phone ?: throw JSONException("phone is null"),
+                givenName = givenName ?: "",
+                surname = surname ?: "",
```

Comment:
Would it be good to throw exception for the user by initialize fields to empty string  var givenName: String? = ""

Reply:
The demo app seems to have a successful flow when these fields are empty, so I'm not sure if we'd want to throw any exceptions here. The constructor is internal to the SDK as the merchant does not instantiate this class.